tests/argv/host/host.c

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+#include <openenclave/internal/tests.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "argv_u.h"
+
+int main(int argc, const char* argv[])
+{
+    oe_result_t result;
+    oe_enclave_t* enclave = NULL;
+    const oe_enclave_type_t type = OE_ENCLAVE_TYPE_SGX;
```

Comment:
Consider changing `OE_ENCLAVE_TYPE_SGX` to `OE_ENCLAVE_TYPE_AUTO` so that the test may (eventually) run on TrustZone as well, seeing as the functionality is not SGX-specific.

Reply:
Thanks. Resolved.